### ISSUE TITLE
How can I check my submission?

### ISSUE THREAD

**Issue by han-minhee** [snu-sf-class/pl201602#17]
(I guess that) I solved the problems in the assignment,
but how can I check whether I've done right or not?

README says
"Both make and make eval SHOULD SUCCEED. If not, your score will be 0.
make eval will check your submission.
P??.v files SHOULD NOT contain admit, Admitted, and anything in forbidden.txt.
If a P??.v file contains string FILL_IN_HERE, then it will be scored 0.
"

but even if I try make eval, I don't understand what they say.
It seems that they succeeded, but I don't know if they checked my submission or whatever.
Could you explain a little bit more?

**Comment by jeehoonkang** [comment]
- what does `make eval` say?
- just upload your submission to the evaluation server, and see what it says.

**Comment by han-minhee** [comment]
I just uploaded it to the server and it shows SUCCESS!
Thank you!

However, make or make eval shows exactly same message regardless of whether I solved the problems or not.
It still shows the same message that I got when I first downloaded the files and tried make, make eval.
What is it supposed to say?

**Comment by jeehoonkang** [comment]
Success dosnt necessarily mean you got the full score.  Look at the score for more information.

Make eval is expected to emit no error message.  If it doesnt emit one and you didnt use fillinhere, then you will get the full score.

**Comment by han-minhee** [comment]
Do I get full score if the server shows 50/50?

And as I checked it now, it shows different message in terminal.
I guess I did something wrong before.
For the unsolved problems, if I try make eval, it says
`FILL_IN_HERE:
P01.v:  FILL_IN_HERE.
P01.v:Proof. exact FILL_IN_HERE. Qed.
P01.v:Proof. exact FILL_IN_HERE. Qed.
P02.v:  FILL_IN_HERE.
P02.v:Proof. exact FILL_IN_HERE. Qed.
P02.v:Proof. exact FILL_IN_HERE. Qed.
P02.v:Proof. exact FILL_IN_HERE. Qed.
P03.v:  exact FILL_IN_HERE.
P04.v:  exact FILL_IN_HERE.
P05.v:  exact FILL_IN_HERE.
`
and they don't show up for the solved ones.
Sorry to bother you. It was just my mistake.
